Stack/Opc.Ua.Core/Types/Encoders/BinaryDecoder.cs

```diff
@@ -2075,6 +2079,8 @@ private ExtensionObject ReadExtensionObject()
 
                             // update body.
                             extension.Body = body;
+
+                            xmlDecoder.Close();
```

Comment:
why is this call necessary, the dispose after the using block calls the same code?

Reply:
its just for completeness .. the encoders/decoders need to unify how close/dispose is handled .. part of the next fuzzing update